changelog/6725.bugfix.rst

```diff
@@ -0,0 +1 @@
+XRT Maps now detect and include the unit property.
```

Comment:
```suggestion
`.XRTMap` now will set the units for level 0 and level 1 files correctly.
```

Reply:
This unit checking should work for any processing level, not just 0 or 1. Also, I don't think that intersphinx link is right.